src/modules/nRF24/nRF24.cpp

```diff
@@ -438,6 +438,13 @@ size_t nRF24::getPacketLength(bool update) {
 }
 
 int16_t nRF24::setCrcFiltering(bool crcOn) {
+  // Auto Ack needs to be disabled in order to disable CRC.
```

Comment:
Enable CRC is forced high if one of the bits in EN_AA is high (datasheet, page 54)